src/components/CustomerDetails/ManualPassportInput.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState, FunctionComponent, useEffect } from "react";
+import { StyleSheet, View } from "react-native";
+import { InputWithLabel } from "../Layout/InputWithLabel";
+import { size } from "../../common/styles";
+import { DarkButton } from "../Layout/Buttons/DarkButton";
+import { DropdownFilterInput } from "../DropdownFilterModal/DropdownFilterInput";
+import { nationalityItems } from "../DropdownFilterModal/nationalityItems";
+import { DropdownItem } from "../DropdownFilterModal/DropdownFilterModal";
+
+const styles = StyleSheet.create({
+  centeredView: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center"
+  },
+  inputAndButtonWrapper: {
+    marginTop: size(3),
+    flexDirection: "row",
+    alignItems: "flex-end"
+  },
+  inputWrapper: {
+    flex: 1,
+    marginRight: size(1)
+  }
+});
+
+interface ManualPassportInput {
+  setIdInput: (id: string) => void;
+  submitId: () => void;
+}
+
+export const ManualPassportInput: FunctionComponent<ManualPassportInput> = ({
+  setIdInput,
+  submitId
+}) => {
+  const [selectedCountry, setSelectedCountry] = useState<DropdownItem | null>(
+    null
+  );
+  const [passportNo, setPassportNo] = useState<string | null>(null);
+
+  const onItemSelection = (item: DropdownItem): void => {
+    setSelectedCountry(item);
+  };
+
+  const onPassportNoChanged = (passportNo: string): void => {
```

Comment:
Minor nit, shall we use `num` rather than `no`, keep reading it as "no changed" 😅

Reply:
done. Change to `num`